source/_integrations/nws.markdown

```diff
@@ -25,6 +25,8 @@ According to the [API documentation](https://www.weather.gov/documentation/servi
 
 Providing a METAR station code is optional, and if not supplied, the closest station to the latitude and longitude will be chosen. A list of nearby stations is printed to the log with level `DEBUG` if no station is supplied. Stations can also be found on the [NOAA website](https://www.cnrfc.noaa.gov/metar.php). Codes with only three characters, for example, `ADW` should be prefixed with the letter K, `KADW`.
 
-Two weather entities are created for each entry in the configuration: one for day and night forecasts and one for hourly forecasts. The hourly forecast entity is disabled after configuration but can be enabled by the user. The time supplied for each forecast is the start time for the forecast. Sensors are also created as disabled entities after configuration and can be enabled by the user.
+One weather entity is created to provide forecasts for each entry in the configuration. The time supplied for each forecast is the start time for the forecast. Hourly forecast data can be obtained using the `weather.get_forecasts` service. Sensors are also created as disabled entities after configuration and can be enabled by the user.
```

Comment:
What about this:

```suggestion
One weather entity is created for each entry in the configuration. Hourly and day/night forecasts are provided through the `weather.get_forecasts` service. The time supplied for each forecast is the start time for the forecast. Sensors are also created as disabled entities after configuration and can be enabled by the user.

```

Reply:
Technically sounds great, and I'm happy to commit it if you don't want to mention UI visibility.  The reason I mentioned UI visibility is because it might not be implicitly obvious to new users that the default dashboard and entity display already use this service, and that lack of knowledge might deter them from choosing this integration over others.

Reply:
I actually dont know what you mean by this.  This integration to my knowledge is no different than other weather integrations.

Reply:
Other documentation I looked at was less thorough, and it just seemed to me like reading about the service could make it sound like users have to do something special when all I want is forecasts on their dashboard.  It's probably me overthinking this.

Reply:
I'm not sure this belongs in the NWS documentation, since it applies to _all_ integrations with weather entities.  I would look to see if you think the description in the weather integration itself needs to be clarified.